signIn.html

```diff
@@ -96,8 +96,32 @@ <h1>Sign In</h1>
                 <p>Don't Have account?<a href="signup.html" style="text-decoration: none; color:rgba(37, 173, 208, 0.94);"> Create Account</a></p>
 
             </form>
+                        
         </div>
     </div>
 </div>
+<script>
+    async function sendData(){

```

Comment:
here you get the event passed from onsubmit event and prevent the default behavior using event.preventDefault()